### PR TITLE
Add API V2 Examples

### PR DESCRIPTION
**Pull request by shreythecray**
Updating send API example for API V2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,17 +21,30 @@ from trycourier import Courier
 
 client = Courier(auth_token="your-auth-token") #or set via COURIER_AUTH_TOKEN env var
 
-resp = client.send(
-    event="your-event-id",
-    recipient="your-recipient-id",
-    profile={
-        "email": "example@example.com",
-        "phone_number": "555-867-5309"
+resp = client.send_message(
```
Comment (tk26) [inline README.md:24]: do we want to replace the old example or add v2 calls as new examples? I think we should keep old calls for existing customers who already have been using SDK for v1 requests so if they need to read thru README v1 examples are handy.

File: README.md [hunk 2]
```diff
@@ -21,17 +21,30 @@ from trycourier import Courier
...
+    },
+    "channels": {
+			"email": {
+				"providers": ["gmail"]
+			}
+		}
+  }
 )
+
 print(resp['messageId'])
```
Comment (tk26) [inline README.md:48]: also v2 returns `requestId` not `messageId`

File: README.md [hunk 3]
```diff
@@ -52,10 +90,45 @@ resp = client.send(
     data={
       "world": "Python!"
     }
-)
+
```
Comment (tk26) [inline README.md:93]:
```suggestion
)
```

File: README.md [hunk 4]
```diff
@@ -31,12 +33,48 @@ resp = client.send(
...
```
Comment (tk26) [inline README.md:36]: )

Comment (tk26) [reply README.md:36]:
```suggestion
)
```

File: README.md [hunk 5]
```diff
@@ -86,19 +159,39 @@ resp = client.send(
     data={}, # optional
     preferences={}, # optional
     override={} # optional
```
Comment (tk26) [inline README.md:161]:
```suggestion
    override={} # optional
)   
```

File: README.md [hunk 6]
```diff
@@ -86,19 +159,40 @@ resp = client.send(
...
-)
+)
```
Comment (tk26) [inline README.md:162]:
```suggestion
)
```

**Review by tk26 — APPROVED** [review]

**Comment by AntonGsv** [comment]
Please add an example with sending attachments.

**Pull request closed without merging**